### ISSUE TITLE
[Feature Request] AppleScript as a supported language

### ISSUE THREAD

**Issue by georgesnow** [subethaedit/SubEthaEdit#147]
It would be nice to have AppleScript as an available language for syntax highlighting.

**Comment by monkeydom** [comment]
AppleScript is a special beast, as the files usually are not plain text encoded so they need a convert on load and save. Also the built in apple script editor uses a lot of runtime information which currently is not available to SubEthaEdit. 

Just a simple highlighting mode for text based Apple Script could be done though, if that would be helpful. 

What is your specific use case here?

**Comment by georgesnow** [comment]
yup, I had thought about the runtime info and realized that's kind of a pain to make request for the ability to do that. 

I have noticed though if you save an AppleScript with file extension of. ".applescript" it seems to remove the encoding. And some of the plain text editors I have tried can actually manage that better. And FastScripts (which is what I use for running production scripts) intelligently understands what kind of script whether it be and ".scpt" or ".applescript" file extension.
